packages/demo-fake-api/app.js

```diff
@@ -0,0 +1,12 @@
+const jsonServer = require("json-server");
```

Comment:
why demo-fake-api does not use Typescript?

Reply:
What would be the point to use TS and add a build step for a fake server?